menas/ui/components/dataset/datasetDetail.controller.js

```diff
@@ -408,6 +408,21 @@ sap.ui.define([
           });
         }
 
+        let additionalColumns = oContext.getProperty("additionalColumns");
+        let aOutputColumns = [];
+        for (let key in additionalColumns) {
+          aOutputColumns.push({
+            outputColumn: key,
+            targetAttribute: additionalColumns[key],
+            mappingTableName: oContext.getProperty("mappingTable")
+          });
+        }
+        aOutputColumns.unshift({outputColumn: oContext.getProperty("outputColumn"),
```

Comment:
Same as above. Why to use `unshift` instead of initializing the `aOutputColumns` with the values (line 412)?